Java/measureeval/src/main/java/com/lantanagroup/link/measureeval/configs/SecurityConfig.java

```diff
@@ -12,29 +12,26 @@
 import org.springframework.security.config.annotation.web.builders.HttpSecurity;
 import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
 import org.springframework.security.web.SecurityFilterChain;
-import org.springframework.beans.factory.annotation.Autowired;
 
 
 @Configuration
 @ComponentScan(basePackages = "com.lantanagroup.link.shared.auth")
```

Comment:
I think we might need to also add com.lantanagroup.link.shared.config in basePackages to find the config files.

Reply:
Are you referring to MeasureEvalApplication class cause I do not see @ConfigurationPropertiesScan("com.lantanagroup.link.shared.config") there